features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -35,10 +36,11 @@ internal class DefaultImageWireframeHelper(
     private val imageTypeResolver: ImageTypeResolver
 ) : ImageWireframeHelper {
 
-    @Suppress("ReturnCount")
+    @Suppress("ReturnCount", "LongMethod")
     @UiThread
     override fun createImageWireframe(
         view: View,
+        imagePrivacy: ImagePrivacy,
```

Comment:
this method is blowing up also with all these arguments

Reply:
maybe we could just try to bundle some arguments by category ?

Reply:
I've opened a [ticket](https://datadoghq.atlassian.net/browse/RUM-5446) to do this refactor in a separate PR